src/main/resources/xslt/jats-preview-xslt/xslt/main/jats-html.xsl

```diff
@@ -2250,12 +2250,7 @@ or pipeline) parameterized.
   </xsl:template>
   
 
-  <xsl:template priority="2" mode="list"
-    match="list[@list-type='simple' or list-item/label]">
-    <ul style="list-style-type: none">
-      <xsl:apply-templates select="list-item"/>
-    </ul>
-  </xsl:template>
+ 
```

Comment:
my knowledge of xslt and the html here is lacking. As long as the html output has been reviewed,
it's good.